source/docs/advanced/features/feature-flags.md

```diff
@@ -0,0 +1,31 @@
+---
+id: feature-flags
+title: Feature Flags
+---
+
+Some modules in front-commerce have feature flags that are used to enable/disable set module.
+
+### Enabling/Disabling a feature
+
+Typically the feature flag is defined inside the folder of the module in a loader called `featureFlagLoader.js`. So for example the feature flag for the wishlist module on magento1 is located at `src/server/modules/magento1/wishlist/featureFlagLoader.js`.
+
+To enable/disable a feature it is sufficient to update/override the `featureFlagLoader.js` file to return `true` (to enable the feature) or `false` (to disable the feature) when the name matches the module's name. For example setting the below in `src/server/modules/magento1/wishlist/featureFlagLoader.js` will disable the wishlist feature
```

Comment:
This is not really helpful for integrators IMO.\
They should have to override it from one of their application module.

This Feature Flag custom loader is implemented a bit differently than usual in the sense that it overrides the default implementation.\
What should also be documented to make it clearer is (I think) the `contextEnhancer` too.

Reply:
updated it and made the `contextEnhancer` the main focus